### DISCUSSION TITLE
TVFocusGuideView behaviour tvOS vs AndroidTV

### DISCUSSION THREAD

**Discussion by wouterds** [react-native-tvos/react-native-tvos#385] [General Discussions]
Hi,

The TVFocusGuideView behaves as I would expect it to work on tvOS, whenever it gets focus it focuses the the element I provided in destinations. However, on AndroidTV, it seems to completely ignore it as if it would not exist. The focus just goes to whatever random UI element is closest to the current focus.

Is this expected? If I understand correctly from the docs it is supposed to work the same?

>_TVFocusGuideView: This component provides support for Apple's UIFocusGuide API **and is implemented in the same way for Android TV**, to help ensure that focusable controls can be navigated to, even if they are not directly in line with other controls._

**Comment by douglowder** [comment]
There is existing example code for `TVFocusGuideView` that works correctly on both platforms. To see what is going wrong, I would need to see your code and try it out.

**Reply by wouterds** [reply]
I'll have a look at the example and compare, I'll get back to you, thanks

**Comment by wouterds** [comment]
**Update:** TVFocusGuideView works the same on Android & tvOS in version 0.69.5-2, I was on version 0.66.3-3 where it didn't.

**Reply by wouterds** [reply]
Actually I have to retract my statement, it works on AndroidTV now but not on tvOS. In any case, I'll investigate tomorrow.

**Comment by douglowder** [comment]
Thanks @wouterds , if you do have a test case where the focus guide is failing on tvOS and working on Android, please share it and I'll take a look!
